app/src/main/res/layout/settings_activity.xml

```diff
@@ -1,9 +1,130 @@
+        app:layout_constraintTop_toBottomOf="@+id/chk_Wires" />
+
+    <CheckBox
+        android:id="@+id/chk_Storage"
+        android:layout_width="165dp"
+        android:layout_height="70dp"
+        android:text="@string/storage"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/chk_theCase" />
+
+    <CheckBox
+        android:id="@+id/chk_MotherBoard"
+        android:layout_width="165dp"
+        android:layout_height="70dp"
+        android:layout_marginTop="68dp"
+        android:text="@string/motherboard"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <CheckBox
+        android:id="@+id/chk_HeatSinkFan"
+        android:layout_width="165dp"
+        android:layout_height="70dp"
+        android:text="@string/heat_sink_fan"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/chk_GraphicsCard" />
+
+    <CheckBox
+        android:id="@+id/chk_Processor"
+        android:layout_width="165dp"
+        android:layout_height="70dp"
+        android:text="@string/processor"
+        app:layout_constraintStart_toStartOf="@+id/chk_MotherBoard"
+        app:layout_constraintTop_toBottomOf="@+id/chk_MotherBoard" />
+
+    <CheckBox
+        android:id="@+id/chk_Memory"
+        android:layout_width="165dp"
+        android:layout_height="70dp"
+        android:text="@string/memory"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/chk_HeatSinkFan" />
+
+    <CheckBox
+        android:id="@+id/chk_Watercooling"
+        android:layout_width="165dp"
+        android:layout_height="70dp"
+        android:layout_marginStart="81dp"
+        android:text="@string/watercooling"
+        app:layout_constraintStart_toEndOf="@+id/chk_Storage"
+        app:layout_constraintTop_toBottomOf="@+id/chk_Memory" />
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="130dp"
+        android:layout_marginTop="11dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/chk_Storage"
+        app:srcCompat="@drawable/_09180744"
+        tools:ignore="ContentDescription" />
+
+    <Button
+        android:id="@+id/btn_Submit"
+        android:layout_width="141dp"
+        android:layout_height="84dp"
+        android:layout_marginStart="135dp"
+        android:layout_marginTop="13dp"
+        android:layout_marginEnd="135dp"
+        android:text="@string/submit"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageView" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
According to designdoc The design in MD built a simple interface, but I didn't completely follow the design in the file, because the interface was too crowded and it was difficult for users to control their hands to accurately locate each checkbox when using the mobile phone, so I only put half of the checkboxes on it.